app/Http/Controllers/PeopleController.php

```diff
@@ -205,6 +205,12 @@ public function update(Request $request, $contactId)
             $contact->twitter_profile_url = null;
         }
 
+        if ($request->input('linkedin') != '') {
```

Comment:
For the sake of consistency I just followed the exact same pattern that the other fields had, as I didn't want to change more lines than needed, should I just trim() all the other field inputs as well?